src/sgame/sg_admin.cpp

```diff
@@ -3888,9 +3891,9 @@ bool G_admin_listplayers( gentity_t *ent )
 		           denied,
 		           canseeWarn ? ( p->pers.hasWarnings ? "^3W" : " " ) : "",
 		           p->pers.netname,
-		           ( registeredname ) ? "(a.k.a. " : "",
-		           ( registeredname ) ? registeredname : "",
-		           ( registeredname ) ? "^*)" : "",
+		           ( registeredname && authed ) ? "(a.k.a. " : "",
```

Comment:
The authed check is not needed here since registeredname can only be non-null if it was true

LGTM besides that

Reply:
Good point, I should have checked this before pushing https://github.com/Unvanquished/Unvanquished/pull/2345/commits/a746161432e213dc3ae5c0bcd1e42adbaee1f559. Now fixed.